File: GoriAPI/DAL/Entities/Position.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace DAL.Entities
{
    public class Position
    {
        [Key]
        public int Id { get; set; }
        [Required]
        public string Name { get; set; }
        public string Description { get; set; }
        public decimal RawPrice { get; set; }
        public decimal RetailPrice { get; set; }
        public decimal PositionVolume { get; set; }
        public PositionCategory PositionCategory { get; set; }
        public List<PositionIngredients> PositionIngredients { get; set; }
        public List<Sales> Sales { get; set; }
    }
}

File: Client/src/app/Models/position.ts
export class Position {
    public id : number;
    public name :string;
    public description:string;
    public rawPrice: number;
    public retailPrice: number;
    public positionVolume: number;
    public positionCategoryId: number;
    constructor(values:Object={}){
        Object.assign(this,values);
    }
}
